subsys/usb/class/audio/audio.c

```diff
@@ -10,3 +10,952 @@
  *
  * Driver for USB Audio device class driver
  */
+
+#include <kernel.h>
+#include <usb/usb_common.h>
+#include <usb/usb_device.h>
+#include <usb_descriptor.h>
+#include <usb/usbstruct.h>
+#include <usb/class/usb_audio.h>
+#include "usb_audio_internal.h"
+
+#include <sys/byteorder.h>
+#include <sys/util.h>
+#include <net/buf.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(usb_audio, CONFIG_USB_AUDIO_LOG_LEVEL);
+
+/* Device data structure */
+struct usb_audio_dev_data_t {
```

Comment:
`struct usb_audio_dev_data` or shorter `usb_audio_data` or `audio_context`

Reply:
Will go for `usb_audio_data` as I want to underline this is audio specific structure.

Reply:
Changed my mind, I think better chose would be `struct usb_audio_dev_data` as simple `usb_audio_data` may suggest its data packet or whatsoever. `dev` addition clarifies a lot.